<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestStencilDialog</class>
 <widget class="QDialog" name="TestStencilDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonSetCameraResolution">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>81</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReqCameraResolution">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>81</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Req</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCameraResolution">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Camera resolution</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxCameraResolution">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>601</width>
     <height>631</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="labelCameraResolution_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Board</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxBoardNo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelCameraResolution_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>MotorCounter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonReqMotorCounter">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Req</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMotorCounter">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99999999</number>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetEnableComparator">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>491</width>
      <height>431</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>EnableComparatorL</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>EnableComparatorH</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ForceON</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="labelCameraResolution_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>491</width>
      <height>28</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Light Term</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetEnableComparator">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>330</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonReqEnableComparator">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>260</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Req</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCameraResolution_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Unit</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxUnitNo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAllOff">
    <property name="geometry">
     <rect>
      <x>511</x>
      <y>570</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>All off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCalc">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>500</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Calc</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCalcA">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>430</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
    <property name="value">
     <number>32</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCalcB">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>460</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99999999</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetMotorCounter">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>720</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonRelay">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>720</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Relay</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
